android/cli/commands/_build.js

```diff
@@ -3683,16 +3683,17 @@ AndroidBuilder.prototype.generateAndroidManifest = function generateAndroidManif
 		}
 	}, this);
 
-	// if the target sdk is Android 3.2 or newer, then we need to add 'screenSize' to
-	// the default AndroidManifest.xml's 'configChanges' attribute for all <activity>
-	// elements, otherwise changes in orientation will cause the app to restart
-	if (this.realTargetSDK >= 13) {
+	if (this.realTargetSDK >= 24 && !finalAndroidManifest.application.hasOwnProperty('resizeableActivity')) {
+		finalAndroidManifest.application.resizeableActivity = true;
+	}
```

Comment:
Let's remove this from "_build.js". We don't want to force developers to use "resizeableActivity" in case they have layout issues when sized too small in a split-screen window... in which case they can opt-out by setting this attribute to false in the "tiapp.xml" file.